hack/dockerfile/install/gometalinter.installer

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+GOMETALINTER_COMMIT=bfcc1d6942136fd86eb6f1a6fb328de8398fbd80
+
+install_gometalinter() {
+	echo "Installing gometalinter version $GOMETALINTER_COMMIT"
+	go get -d github.com/alecthomas/gometalinter
+	cd "$GOPATH/src/github.com/alecthomas/gometalinter"
+	git checkout -q "$GOMETALINTER_COMMIT"
```

Comment:
Why not just use `git -C` here instead of doing a `cd` and then a `git` command?

Reply:
These are copied from the old `install-binaries.sh` file. I'd prefer to optimize them separately.